tests/cli/test_check.py

```diff
@@ -81,3 +81,14 @@ def test_check_branch_wrong_side(capsys, git):
     result = runner.invoke(check, ['-r', git, 'message', 'testbranch..master'])
     assert_that(result.exit_code, equal_to(0))
     assert_that(result.output.split("\n"), equal_to([""]))
+
+
+@skip
```

Comment:
Yes, same as the other tests in that file. Here's why:
https://github.com/helix84/kwalitee/blob/31c309917e08371c4621d420f262bf2ebbd917ab/tests/cli/test_check.py#L39-L40